### DISCUSSION TITLE
Is there any way to see the onion skin of the first frame in the last frame?

### DISCUSSION THREAD

**Discussion by furmanslair** [keiwando/toonsquid#392] [Q&A]
I couldn't speak English very well, so I had to use Google Translate.

hello. I love ToonSquid and I use it a lot. I like to make looping frame animations that aren't very long.
  
But since I'm not very good at making animations, it's always difficult to draw a frame that naturally connects from the last frame to the first frame.

So, after copying the first frame, paste it next to the last frame, draw the last frame, and then delete the pasted first frame. If something changes, I repeat it.

Is there any way to see the onion skin of the first frame in the last frame?
If not, do you have any thoughts or plans for an update?

**Comment by DWrightAnimation** [comment]
You say you're not very good at making animations? I know some resources that can help you: 

- The Animator's Survival Kit by Richard Williams 
- Character Animation Crash Course by Eric Goldberg 

These are great animation books, they're both available on Apple Books, and The Animator's Survival Kit has an app with animated examples. 

Also, OpenToonz has what you're looking for, being able to create an onion skin between the first and last frame, however, it's on PC and Mac, but it would be nice to have that feature implemented, being able to onion skin any frames you want, no matter how far apart they are.

**Comment by Cat-Butler** [comment]
Twisted workaround, assuming you are using frame-by-frame animation and not translational/path/switch keyframes: 
Create the first frame as a symbol (in the Library) before inserting it in the main timeline and copying it to the last frame. 

Then if you do end up making changes to the first frame (edit it in the Library, not the main timeline), you don't have to copy it again - changes will automatically show up in both the first frame and the last frame (because that's how symbols work). 

Only delete (if required) the last frame when your animation is complete and ready to export.
